configs/peer/update_legacy_configs.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/sh
-# This script is intended for release updates,
-# when LTS and Stable branch configurations may change.
-# It downloads the previous configurations from raw.githubusercontent.com.
```

Comment:
this was wrong because docker image ID would would be pointing to `hyperledger/iroha2:lts` but configs could be from previous lts or current lts. It would depend on whether the current release is becoming lts.